bazel/scripts/check_py_bazel.sh

```diff
@@ -45,7 +45,6 @@ DENY_LIST_NOT_RELEVANT=(
   "./protos"
   "./show-tech"
   "./third_party"
-  "./xwf/gateway/deploy"
```

Comment:
can we remove this already before #13545 is merged

Reply:
Removing the line while the folder still exists should lead to failing bazel workflows on PRs. So we should keep it an remove it in #13545.

Reply:
ok moved it there